### PR TITLE
create solution and add projects

### PR DESCRIPTION
**Pull request by Andrei-Ha**
create solution

### CODE REVIEW

File: src/BL/ResponseBuilder.cs [hunk 1]
```diff
@@ -0,0 +1,46 @@
+﻿using Exadel.Forecast.BL.Validators;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Exadel.Forecast.BL
+{
+    public class ResponseBuilder
+    {
+        private readonly TemperatureValidator _temperatureValidator;
```
Comment (MaximGomon) [inline src/BL/ResponseBuilder.cs]: Change to IValidator

File: src/DAL/Models/OpenWeatherDayModel.cs [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        public Clouds clouds { get; set; }
+        public int dt { get; set; }
+        public Sys sys { get; set; }
+        public int timezone { get; set; }
+        public int id { get; set; }
+        public string name { get; set; }
+        public int cod { get; set; }
+    }
+
+    public class Coord
```
Comment (MaximGomon) [inline src/DAL/Models/OpenWeatherDayModel.cs]: One file - one type

File: src/DAL/RequestSender.cs [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+using System.IO;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace Exadel.Forecast.DAL
+{
+    public class RequestSender<T>
+    {
+        static readonly HttpClient _httpClient = new HttpClient();
+        public static async Task<T> GetModel(string webUrl)
```
Comment (MaximGomon) [inline src/DAL/RequestSender.cs]: Why static?

File: src/DAL/RequestSender.cs [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+        {
+            try
+            {
+                HttpResponseMessage response = await _httpClient.GetAsync(webUrl);
+                response.EnsureSuccessStatusCode();
+                using (var responseStream = await response.Content.ReadAsStreamAsync())
```
Comment (MaximGomon) [inline src/DAL/RequestSender.cs]: You could directly read response to string

**Comment by Andrei-Ha** [comment]
I will create new PR

**Pull request closed without merging**